src/coreclr/nativeaot/Runtime/unix/UnwindHelpers.cpp

```diff
@@ -883,3 +883,29 @@ bool UnwindHelpers::GetUnwindProcInfo(PCODE pc, UnwindInfoSections &uwInfoSectio
     uc.getInfo(procInfo);
     return true;
 }
+
+#if defined(TARGET_APPLE)
+bool _dyld_find_unwind_sections(void* addr, dyld_unwind_sections* info)
```

Comment:
should we upstream this to llvm-libunwind? I saw a bunch of reports that this is causing problems for people wanting to use it on iOS.

Reply:
maybe also add a comment that we're reimplementing this function

Reply:
LLVM actually had a very similar code in the past as fallback for macOS <10.7, and they removed it. Apple distributes libunwind (the LLVM version) as part of public API in the iOS SDK, so apps are supposed to consume it through `libunwind.h` and `unw_*` APIs, not as source copy. Unfortunately we cannot directly use that API without some significant refactoring with negative performance implications.

> maybe also add a comment that we're reimplementing this function

Can do that.